package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@tootsuite/mastodon",
   "license": "AGPL-3.0-or-later",
   "engines": {
-    "node": ">=8.12 <13"
+    "node": ">=10.13 <13"
```

Comment:
The reason I set the lower limit of node version to 8.12 is to match the lower limit version of the package being used. I think it is better to set it to 10 if there is no special reason to specify the minor version.

Reply:
Similar reasons for mine. #12521 is 10.13 for that package.

Reply:
ok, I understand.